libs/product/src/models/composite-product-item.ts

```diff
@@ -0,0 +1,17 @@
+import { DaffProduct } from './product';
+
+export interface DaffCompositeProductItem {
+	id: number;
```

Comment:
I think we should make this `number | string`

Reply:
done